### PR TITLE
Light theme accessibility styling fixes

### PR DESCRIPTION
**Pull request by thibaudcolas**
Preamble to #11 – covers accessibility and print styling fixes for elements that aren’t completely changed in #58. Specifically:

- Better support for font resizing – switching dimensions from pixels to `rem` where appropriate, with loosely similar sizes at default browser font sizes.
- Support for forced colors mode / Windows high contrast mode / Contrast themes. With a few elements completely hidden, a few additional borders to help identify parts of the page that otherwise rely on backgrounds, and high-contrast filters for specific images.
- Print stylesheet tweaks

Tested in latest Safari, Firefox, Chrome on macOS 13.1.

## Font resizing

To test this, I’d recommend changing the browser font size to "Very large" in Chrome settings. Screenshot:

font-very-large

There are a few elements that still have issues (GB map background, footer logo) but I’m changing them much further in #58 anyway.

## Forced colors mode

To test this, the simplest is to use "Emulate forced-colors: active" in the Chrome DevTools. Screenshot:

whcm

Again some parts of the page don’t work so well (dates section background, footer curve) but I’m changing them in #58 anyway.

## Print stylesheets

I only did minimal adjustments so the site would print a bit better. I’ve done quite a bit more in #58.

### CODE REVIEW

**Comment by sabderemane** [comment]
hey @thibaudcolas, looks good to me, I've just seen this, I guess it's part of the change in the draft PR ?
Capture d’écran 2023-03-15 à 12 13 38

this is for the font resize

**Review by sabderemane — COMMENTED** [review]
hey @thibaudcolas, looks good to me, I've just seen this, I guess it's part of the change in the draft PR?
Capture d’écran 2023-03-15 à 12 13 38
this is for the font resize

File: _src/scss/layout/_navbar.scss [hunk 1]
```diff
@@ -5,7 +5,8 @@
 // title bar text overrides
 .title-bar .menu a {
   text-decoration: none;
-  color: $white;
+  // Force links to always be white even on hover.
+  color: $white !important;
```
[the remark this answers is not in the document]
Comment (sabderemane) [reply _src/scss/layout/_navbar.scss:9]: it's not in [settings.scss](https://github.com/djangocon/2023.djangocon.eu/blob/a3a0c2e04053b9a8b0a7aae667c07306092f1d9e/_src/scss/_settings.scss#L92) file ? 
I've seen definition of blue colors at the top and then Foundation colors but I'm not completely familiar with Foundation.

[the remark this answers is not in the document]
Comment (thibaudcolas) [reply _src/scss/layout/_navbar.scss:9]: It’s not, it’s the `primary` color from Foundation. I thought it was meant to be overridden by our settings’ `$foundation-palette` but this doesn’t seem to be working.

[the remark this answers is not in the document]
Comment (sabderemane) [reply _src/scss/layout/_navbar.scss:9]: Weird, got it, thanks for the explanation!

**Comment by thibaudcolas** [comment]
@sabderemane good catch! I’m not sure whether I’ve tackled this already but will make sure to.

**Pull request merged**